database-compose.yml

```diff
@@ -1,9 +1,11 @@
 #SPDX-License-Identifier: MIT
-#Format for ports: <outside>:<inside>
 version: '3'
 services:
    database:
     image: augurlabs/augur:${AUGUR_DB_TYPE}
+    build:
+      context: .
+      dockerfile: ./util/docker/database/Dockerfile
     restart: unless-stopped
     ports:
-     - 5434:5432
\ No newline at end of file
+     - 5434:5432
```

Comment:
Why is that exposed as `5434` on the host? would it be OK to switch to 5432?

Reply:
I believe that is a Docker related port mapping. Since the local Postgres instances typically resolve to 5432, or 5433, the Docker container (I think) uses 5434 to avoid a conflict. However, it is possible I misunderstand. In our use case, we are running the Docker container on the same metal as the database. @maxhbr

Reply:
yes, it is probably for avoiding conflicts. But it creates other conflicts by having to change config (either talk to 5432 or 5434)